### ISSUE TITLE
1062 Duplicate entry for key 'name' SQL=INSERT INTO

### ISSUE THREAD

**Issue by JKoelman** [joomleague/JoomLeague#44]
Did notice that it's not possible (in some cases) to add duplicate values as it will show a 1062 duplicate entry notice. Think the idea is good but it would be better to make the notice more friendly.

**Comment by CRillaerts** [comment]
Yes I noticed it too, I tried to add a season with the same name: it was not accepted

**Comment by JKoelman** [comment]
did add a check to a few table files, the rest will be for later.
